crdb/crdbgorm/gorm_test.go

```diff
@@ -32,12 +35,12 @@ func TestExecuteTx(t *testing.T) {
 	}
 	ctx := context.Background()
 
-	gormDB, err := gorm.Open("postgres", ts.PGURL().String())
+	gormDB, err := gorm.Open(postgres.Open(ts.PGURL().String()), &gorm.Config{})
 	if err != nil {
 		t.Fatal(err)
 	}
 	// Set to true and gorm logs all the queries.
```

Comment:
the comment should say "Set to logger.Info and gorm logs all the queries"

Reply:
thanks for the comment!